### ISSUE TITLE
Can't get Keypair from address without mnemonic or seed

### ISSUE THREAD

**Issue by AlaaElattar** [codescalers/tfgrid-sdk-dart#14]

**Comment by zaelgohary** [comment]
Blocked on Polkadart package.

**Comment by xmonader** [comment]
But we should not be able to get the key pair (including the private key) from just the address, right ?

**Comment by AhmedHanafy725** [comment]
> But we should not be able to get the key pair (including the private key) from just the address, right ?

true, it just loads the address and enable the features that can be done with the address only like verifying a signature
